### PR TITLE
Migrate to shaclex 0.0.76

### PR DESCRIPTION
**Pull request by berezovskyi**

### CODE REVIEW

**Comment by berezovskyi** [comment]
All cardinality based tests seem to fail

File: pom.xml [hunk 1]
```diff
@@ -13,8 +13,8 @@
 
 		<version.jetty>9.4.0.v20161208</version.jetty>
 		<version.lyo>2.4.0-SNAPSHOT</version.lyo>
-		<version.shaclex>0.0.63</version.shaclex>
-		<jena.version>3.0.1</jena.version>
+		<version.shaclex>0.0.76</version.shaclex>
+		<jena.version>3.6.0</jena.version>
```
Comment (yashkhatri) [inline pom.xml]: Are we planning to move from Jena 3.0.1 to 3.6.0 in all Lyo Projects? Was there a need to upgrade Jena Version here?

Comment (berezovskyi) [reply pom.xml]: There is no way to upgrade Jena in one project and not another. Then lyo-store 2.4.0 would be incompat with lyo-validation 2.4.0...

**Comment by berezovskyi** [comment]
-1 needs CQs

**Comment by berezovskyi** [comment]
Great refactoring, @yashkhatri!

File: model/shacl_domain.xml [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+    </resourceProperties>
+    <resourceProperties title="Node Kind" name="nodeKind" readOnly="true" valueType="URI" description="Specifies the node kind. Values can be: sh:BlankNode, sh:IRI, sh:Literal, sh:BlankNodeOrIRI, sh:BlankNodeOrLiteral and sh:IRIOrLiteral"/>
+    <generationSetting javaClassPackageName="" javaFilesPath=""/>
+  </domainSpecifications>
+  <domainSpecifications name="Java" namespaceURI="http://your.organisation.domain/nsp2#" namespacePrefix="//@domainPrefixes.1">
+    <resources name="Object"/>
+    <generationSetting javaClassPackageName="java.lang" javaFilesPath=""/>
+  </domainSpecifications>
+  <domainPrefixes name="sh"/>
+  <domainPrefixes name="nsp2"/>
```
Comment (berezovskyi) [inline model/shacl_domain.xml:73]: strange

**Pull request merged**